docs/HelpWantedIssueSummaryCommentTemplate.md

```diff
@@ -0,0 +1,33 @@
+<!--
+
+**IMPORTANT**: As part of posting this comment, please make sure:
+- that the issue is properly labeled with both `area-` and `feature-` labels.
+- to apply one of the `Complexity: ` labels is applied to the issue [Simple | Medium | Hard]
+- for an issue which was marked with `Complexity: Simple` label, it may be a good fit for a first-time contributors to this repo.
+  If so, please also apply `good-first-issue` label.
+-->
+
+## Issue Summary
+
+<!-- This section will be filled in by the engineers assigned to this  -->
+
+## Potential Design
+
+<!-- This section is meant to be used for a high level design of what a solution should look like and/or what direction should be taken to solving this problem. -->
+
+## Code References
+
+<!-- Links to classes / methods which are critical to understand for building out this solution, as these are relevant and will be used by the solution, as well as existing test cases that exercise this code -->
+
+## Relevant Documentation
+
+<!-- Links to relevant documentation which you think is relevant for handling this issue -->
+
+## Important Considerations
+
+<!-- List of additional gotchas the community member who will be handling this issue will need to consider when working on the solution -->
+
+## Additional Notes
+- If the change you are going to make will include a **public API change** in the framework, then the change will need to go through our [API Review Process](https://github.com/dotnet/aspnetcore/blob/main/docs/APIReviewProcess.md). Please read this process to take any necessary steps, to avoid any potential delays down the road with accepting this change.
```

Comment:
```suggestion
- If the change you are going to make will include a **public API change** in the framework, then the change will need to go through our [API Review Process](/docs/APIReviewProcess.md). Please read this process to take any necessary steps, to avoid any potential delays down the road with accepting this change.
```